app/services/book.py

```diff
@@ -0,0 +1,22 @@
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
+        if display_type == "console":
+            print(self.content)
+        elif display_type == "reverse":
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")
+
+    def print_book(self, print_type: str) -> None:
+        if print_type == "console":
+            print(f"Printing the book: {self.title}...")
+            print(self.content)
+        elif print_type == "reverse":
+            print(f"Printing the book in reverse: {self.title}...")
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown print type: {print_type}")
```

Comment:
It looks like your Book class does a few things. Let's follow single responsibility principle